content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,468 @@
+To be able to receive invoices and notifications, the :abbr:`SdI (Sistema di Interscambio)` service
+must be notified that the user's files are to be sent to **Odoo**, and are processed on their
+behalf. To do so, you must set up Odoo's :guilabel:`Codice Destinatario` on the **Agenzia Delle
+Entrate** portal.
 
-#. Go to https://ivaservizi.agenziaentrate.gov.it/portale/ and authenticate.
-#. Go to section :menuselection:`Fatture e Corrispettivi`.
-#. Set the user as Legal Party for the VAT number you wish to configure the electronic adress.
+#. Go to https://ivaservizi.agenziaentrate.gov.it/portale/ and authenticate;
+#. Go to section :menuselection:`Fatture e Corrispettivi`;
+#. Set the user as Legal Party for the VAT number you wish to configure the electronic address;
 #. In :menuselection:`Servizi Disponibili --> Fatturazione Elettronica --> Registrazione
-   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, input Odoo's Codice
-   Destinatario (**K95IV18**), then confirm.
+   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, insert Odoo's
+   :guilabel:`Codice Destinatario` `K95IV18`, and confirm.
 
-Give Odoo permission to process files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Electronic Data Interchange (EDI)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Since the files are transmitted through Odoo's server before being sent to SDICoop or received by
-your database, you need to authorize Odoo to process your files from your database.
+Odoo uses the **FatturaPA** :abbr:`EDI (Electronic Data Interchange)` format for the Italian
+localization, and is enabled by default when installed. When the **file processing authorization**
+has been set, all **invoices** and **bills** are automatically sent for journals with
+:guilabel:`Electronic invoicing` enabled (:menuselection:`Accounting --> Configuration --> Journals
+--> (Select a journal) --> Advanced Settings --> Electronic Data Interchange)`.
 
-To do this, go to :menuselection:`Accounting --> Settings --> Electronic Invoicing` and click on
-*Register*.
+You can check the current status of an invoice by the :guilabel:`Electronic invoicing` field *(cf.
+screenshot below)*. The XML file can be found in the **chatter** of the invoice.
 
-.. image:: italy/register.png
+.. image:: italy/italy-processing.png
    :align: center
-   :alt: Register button to allow Odoo to process invoices.
+   :alt: Electronic invoicing status (waiting for confirmation)
 
-.. Note::
-   All your files are encrypted upon reception in such a way that only you are able to decrypt them.
+.. seealso::
+   :doc:`../../receivables/customer_invoices/electronic_invoicing`
 
-Enable FatturaPA on the Sales journal
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+File processing authorization (Odoo)
+------------------------------------
+
+Since the files are transmitted through Odoo's server before being sent to SDICoop or received by
```

Comment:
```suggestion
Since the files are transmitted through Odoo's server before being sent to the SDI or received by
```
SDICoop is the name of the integration with the SdI system, but this information is unnecessary to the user. Just use `SdI` that you already described.